### DISCUSSION TITLE
MAX1704X Component

### DISCUSSION THREAD

**Discussion by besbar** [UncleRus/esp-idf-lib#530] [General]
Hi, just a general question about components listed in this repo.

I downloaded few days ago the MAX1704X component (LiPo battery gauge) which was not in the Master branch.
Today I cannot find this branch where the MAX1704x was available, and I cannot find it neither in Master.
It is no problem for me as I have downloaded and tested the code successfully (though one or two features are not properly working).

But I am curious to know why this component is not yet included in the master branch (still beta version of this component?).

Thanks.

**Comment by besbar** [comment]
Sorry actually this component is avalaible in this repo forked by Shuki25 (https://github.com/shuki25/esp-idf-lib/tree/feature-max1704x).

**Comment by UncleRus** [comment]
@shuki25 How about making a PR? :)
